nucliadb/nucliadb/writer/tests/unit/test_field.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2021 Bosutech XXI S.L.
```

Comment:
should this file be in the `unit/resources/test_field` so we can mirror the full mod path?